api/api.go

```diff
@@ -63,6 +63,8 @@ const (
 	SpecExportOptionsEmpty   = "empty_export_options"
 	SpecMountOptions         = "mount_options"
 	SpecCSIMountOptions      = "csi_mount_options"
+	SpecCSIRawBlock			 = "fadirectRawBlock"
+	SpecCSIFsType			 = "fsType"
```

Comment:
we already have a `fs` spec - see above `	SpecFilesystem           = "fs"`

Can we reuse that one?

Reply:
Yes we should reuse the `fs` parameter. Also the other parameter `CSIRawBlock` is not really required. The CSI driver in openstorage already handles Raw Block Devices. When RawBlock PVCs are used the spec.Format is set to `api.FSType_FS_TYPE_NONE`